README.md

```diff
@@ -0,0 +1,499 @@
+})
+
+app.listen(port, () => {
+  console.log(`Tailwind as a service listening on port ${port}`)
+})
+```
+
+Running the server with node we have the response directly in the browser:
+
+```bash
+$ node ./src/tailwind-as-a-service.js
+```
+
+![Hello World](images/hello-world.png)
+
+> We are using Node 16, which supports [ES modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) by default. If you are using older versions of Node you can use CommonJs modules and just rename the file with the `.cjs` extension.
+>
+
+So far, so good. Now we are going to configure `postcss` and its tailwind plugin to return CSS:
+
+```jsx
+// src/tailwind-as-a-service.js
+......
+import postcss from 'postcss';
+import tailwindcss from 'tailwindcss';
+
+......
+
+const defaultCss = `
+  @import 'tailwindcss/base';
+  @import 'tailwindcss/components';
+  @import 'tailwindcss/utilities';
+`;
+
+app.get('/', async (req, res) => {
+  const configuredTailwind = tailwindcss({
+    content: [{ raw: '<div class="bg-red-500">', extension: 'html' }]
+  });
+  const postcssProcessor = postcss([configuredTailwind]);
+  const { css } = await postcssProcessor.process(defaultCss);
+  res.send(css);
+});
+
+......
+```
+
+We added the `postcss` and `tailwindcss` dependencies. Then we configure the Tailwindcss plugin for postcss with the [content](https://tailwindcss.com/docs/content-configuration) option.
+
+This option tells Tailwindcss to inspect the HTML, JavaScript components and more files, to look for CSS classes to generate and include its CSS in the final result. It allows also us to put inline “raw” HTML.
+
+After that, we create a `postcssProcessor` with the configured plugin of Tailwindcss. This is responsible for parsing the CSS and applying all postcss plugins.
+
+Finally, we process a “fake” CSS file with the default base, components and utility styles of Tailwindcss. This is necessary to make Tailwindcss generate all necessary CSS.
+
+The result CSS is returned in the response. So if we run again the service with `node ./src/tailwind-as-a-service.js` and we request from the browser, then we can see the resulting CSS:
+
+![CSS response](images/css-response.png)
+
+Here you can see the base CSS that [Tailwindcss brings by default](https://tailwindcss.com/docs/preflight) and also, at the end, the `.bg-red-500` class that we are passing as raw HTML to the Tailwindcss config.
+
+So we have a service to request and return the CSS, but how do we configure that CSS? well let’s make this service receive parameters and use them to configure the Tailwindcss plugin:
+
+```jsx
+// src/tailwind-as-a-service.js
+
+......
+app.post('/', async (req, res) => {
+  const configuredTailwind = tailwindcss({
+    content: [{ raw: req.body.html, extension: 'html' }],
+    theme: req.body.theme
+  });
+......
+});
+......
+```
+
+Here, we changed the `.get` method into `.post` to be able to send and receive parameters in the request body, for larger parameters.
```

Comment:
Added some missing lines to make it clearer. 

BTW the `-` didn't work for me, it breaks the markdown.

Reply:
Oh, sorry, I meant I included it to avoid github's autoformatting, so you could copy paste it directly, instead of seeing the md part of the comment formatted and having to rewrite it by yourself. Sorry if I didn't explained myself clearly and you ended up wasting time 🤦🏿